### PR TITLE
[Fix] no-unknown-property: No unknown property doesn't recognize svg properties

### PR DESCRIPTION
**Pull request by AhmadMayo**
fixes https://github.com/jsx-eslint/eslint-plugin-react/issues/3380

I don't know if the added attributes are exhaustive or not. I sure hope MDN covered all possible attributes.

### CODE REVIEW

File: lib/rules/no-unknown-property.js [hunk 1]
```diff
@@ -29,6 +29,9 @@ const DOM_ATTRIBUTE_NAMES = {
 const ATTRIBUTE_TAGS_MAP = {
   // image is required for SVG support, all other tags are HTML.
   crossOrigin: ['script', 'img', 'video', 'audio', 'link', 'image'],
+  fill: ['svg'],
+  property: ['meta'],
+  viewBox: ['svg'],
```
Comment (ljharb) [inline lib/rules/no-unknown-property.js:34]: i added these as well.

**Review by ljharb — APPROVED** [review]
Looks great!

File: lib/rules/no-unknown-property.js [hunk 2]
```diff
@@ -29,6 +29,9 @@ const DOM_ATTRIBUTE_NAMES = {
 const ATTRIBUTE_TAGS_MAP = {
   // image is required for SVG support, all other tags are HTML.
   crossOrigin: ['script', 'img', 'video', 'audio', 'link', 'image'],
+  fill: ['svg'],
```
Comment (shepmaster) [inline lib/rules/no-unknown-property.js:32]: `fill` appears to be OK [on a number of other tags](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill), such as `path`

Comment (ljharb) [reply lib/rules/no-unknown-property.js:32]: This should be handled by #3385

**Pull request merged**